### DISCUSSION TITLE
Turning Kin to a deflationary currency

### DISCUSSION THREAD

**Discussion by skyf0xx** [kinfoundation/kin-rfcs#177] [KRE 4.0 Discussion]
Proposal here: https://github.com/kin-labs/rewards-engine/blob/master/improvement-proposals/wg_kre_inflation.md

**Comment by mocolicious** [comment]
I like the idea of capping to address the issue about having a fixed amount and rewards tending to centralize to the top.  The top 10 apps earn something like 99% of the KRE (don't quote me reference https://kinscan.io chart).  I think that would solve the problem of apps not wanting to collaborate because it would take away from their own earnings.  I believe that we should and will bring more apps into the ecosystem because it will leave more kin up for grabs for newcomers and apps can work together to elevate each other.  The economy will be created by many people reaping this benefit.

In addition to that there must be a way that sell pressure can be offset by allocating kin towards something like yield farm incentives where an equal amount of kin can be removed from circulation temporarily.  

It needs to be deflationary if people are going to hodl it, without achieving that it will always be a token that people immediately dump.  At the very least it needs to inflate similarly or less than the US dollar.
